tests/eko/test_kernels_s.py

```diff
@@ -149,6 +149,52 @@ def test_zero_nnlo_decompose(monkeypatch):
             pass
 
 
+def test_zero_n3lo_decompose(monkeypatch):
+    """No evolution results in exp(0)"""
+    nf = 3
+    ev_op_iterations = 3
+    ev_op_max_order = 3
+    gamma_s = np.random.rand(4, 2, 2) + np.random.rand(4, 2, 2) * 1j
+    monkeypatch.setattr(
+        ad,
+        "exp_singlet",
+        lambda gamma_S: (
+            gamma_S,
+            1,
+            1,
+            np.array([[1, 0], [0, 0]]),
+            np.array([[0, 0], [0, 1]]),
+        ),
+    )
+    for method in [
+        "decompose-expanded",
+        "decompose-exact",
+    ]:
+        try:
+            np.testing.assert_allclose(
+                s.dispatcher(
+                    (4, 0), method, gamma_s, 1, 1, nf, ev_op_iterations, ev_op_max_order
+                ),
+                np.zeros((2, 2)),
+                atol=1e-15,
+            )
+            np.testing.assert_allclose(
+                s.dispatcher(
+                    (4, 0),
+                    method,
+                    np.zeros((4, 2, 2), dtype=complex),
+                    2,
+                    1,
+                    nf,
+                    ev_op_iterations,
+                    ev_op_max_order,
+                ),
+                np.zeros((2, 2)),
+            )
+        except ZeroDivisionError:
```

Comment:
Why is this happening?

Reply:
This is copied from the previous orders, I haven't checked why it's there honestly...

Reply:
This is copied from the previous orders, I haven't checked why it's there honestly...